templates/repo/home.tmpl

```diff
@@ -4,171 +4,222 @@
-							{{- end -}}
-						{{- end -}}
-					</span>
-				{{end}}
-			</div>
-			<div class="repo-button-row-right">
-				<!-- Only show clone panel in repository home page -->
-				{{if $isHomepage}}
-					<div class="clone-panel ui action tiny input">
-						{{template "repo/clone_buttons" .}}
-						<button class="ui small jump dropdown icon button" data-tooltip-content="{{ctx.Locale.Tr "repo.more_operations"}}">
-							{{svg "octicon-kebab-horizontal"}}
-							<div class="menu">
-								{{if not $.DisableDownloadSourceArchives}}
-									<a class="item archive-link" href="{{$.RepoLink}}/archive/{{PathEscapeSegments $.RefName}}.zip" rel="nofollow">{{svg "octicon-file-zip" 16 "tw-mr-2"}}{{ctx.Locale.Tr "repo.download_zip"}}</a>
-									<a class="item archive-link" href="{{$.RepoLink}}/archive/{{PathEscapeSegments $.RefName}}.tar.gz" rel="nofollow">{{svg "octicon-file-zip" 16 "tw-mr-2"}}{{ctx.Locale.Tr "repo.download_tar"}}</a>
-									<a class="item archive-link" href="{{$.RepoLink}}/archive/{{PathEscapeSegments $.RefName}}.bundle" rel="nofollow">{{svg "octicon-package" 16 "tw-mr-2"}}{{ctx.Locale.Tr "repo.download_bundle"}}</a>
-								{{end}}
-								{{if .CitiationExist}}
-									<a class="item" id="cite-repo-button">{{svg "octicon-cross-reference" 16 "tw-mr-2"}}{{ctx.Locale.Tr "repo.cite_this_repo"}}</a>
-								{{end}}
-								{{range .OpenWithEditorApps}}
-									<a class="item js-clone-url-editor" data-href-template="{{.OpenURL}}">{{.IconHTML}}{{ctx.Locale.Tr "repo.open_with_editor" .DisplayName}}</a>
-								{{end}}
+
+						{{if .LatestRelease}}
+							<div class="flex-item">
+								<div class="flex-item-main">
+									<div class="flex-item-title">
+										<a class="item muted" href="{{.Link}}/releases">
+											{{ctx.Locale.Tr "repo.releases"}}
+											<span class="ui small label">{{.NumReleases}}</span>
+										</a>
+									</div>
+									<div class="flex-item">
+										<div class="flex-item-icon">
+											{{svg "octicon-tag" 16}}
+										</div>
+										<div class="flex-item-main">
+											<div class="flex-item-header">
+												<div class="flex-item-title tw-gap-2">
+													<a class="latest-releases muted" href="{{.LatestRelease.Link}}" data-tooltip-content="{{.LatestRelease.Title}}">{{.LatestRelease.Title}}</a>
+													<span class="ui basic green label tw-h-100">{{ctx.Locale.Tr "latest"}}</span>
+												</div>
+											</div>
+											<div class="flex-item-body">
+												<span class="time">{{DateUtils.TimeSince .LatestRelease.CreatedUnix}}</span>
+											</div>
+										</div>
+									</div>
+								</div>
 							</div>
-						</button>
-						{{template "repo/clone_script" .}}{{/* the script will update `.js-clone-url` and related elements */}}
+						{{end}}
+
+						{{if and (.Permission.CanRead ctx.Consts.RepoUnitTypeCode) (not .IsEmptyRepo) .LanguageStats}}
```

Comment:
Why it needs to check `CanRead RepoUnitTypeCode`?  The "home page" already means that the doer has RepoUnitTypeCode permission?

Reply:
Done in 67026e1d2a7a768e2871f9ff06ed1e9a15cde5d3